docs/docsite/rst/community/steering/community_steering_committee.rst

```diff
@@ -146,13 +146,19 @@ When reviewing community collection `inclusion requests <https://github.com/ansi
 
 #. For a collection to be included in the Ansible community package, the collection:
 
-  * MUST be reviewed and approved by at least two persons, where at least one person is a Steering Committee member.
-  * For a Non-Steering Committee review to be counted for inclusion, it MUST be checked and approved by *another* Steering Committee member.
-  * Reviewers must not be involved significantly in development of the collection. They must declare any potential conflict of interest (for example, being friends/relatives/coworkers of the maintainers/authors, being users of the collection, or having contributed to that collection recently or in the past).
+  * MUST be reviewed and approved as compliant with the requirements by at least two Steering Committee members.
```

Comment:
This is now stricter than the original requirements (one SC person + another community person, which could be SC but doesn't have to be). Is this intentional?

Reply:
I don't think it's practically stricter as now there must be two SC approvals and how we can approve w/o actually reviewing a collection even selectively?

Reply:
Checking a review (and quickly looking at very few files in the collection) is IMO less work than doing a partial review of the collection.

Reply:
@felixfontein i believe it's the same or even less work for the SC member who's checking (who's looking at the code and docs primarily anyway) but there will be no need for the second full review by the third person - so it's actually a significant simplification